README.md

```diff
@@ -43,6 +43,8 @@ Requirements:
 ```sh
 # install npm dependencies and Selenium (for tests)
 make install
+# create database based on params found in `.envrc` file
```

Comment:
Nope, it depends on your env variables. It depends on your `.envrc` if you use direnv